.github/workflows/publish-latest.yml

```diff
@@ -21,9 +24,9 @@ jobs:
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@b98cec97583b917ff1dc6179dd4d230d3e439894
-        # with:
-        # version: yarn version:latest # commented out until we're ready for GA release
-        # publish: yarn release:latest # commented out until we're ready for GA release
+        with:
+          version: yarn version:latest
```

Comment:
If this PR is merged, does this version to latest or publish to NPM? Or is this passing the commands for that PR to run once it's merged?

https://github.com/aws-amplify/amplify-ui/pull/596

Reply:
Good question, this was just to pass the command that `@changesets/action` can use... but this is not actually neede reading through their README: https://github.com/changesets/action#with-version-script

> If the version script is present, this action will run that script instead of changeset version, so please make sure that your script calls changeset version at some point.

I'll remove it from `yml`, but keep the version command in `package.json` just in case we need to publish manually.